### PR TITLE
Add `min` and `max` aliases

### LINKED ISSUE #9: Shorten options
I think it's a massive hog of space to be calling them `minimum` and `maximum`. The shorter `min` and `max` say the exact same thing and are virtually no more confusing.

### PR DESCRIPTION
**Pull request by Richienb**
Fixes #9

### CODE REVIEW

**Comment by Richienb** [comment]
Less code to drop the longer form, but maybe better UX to keep both

File: readme.md [hunk 1]
```diff
@@ -13,19 +13,22 @@ npm install math-clamp
 ```js
 import mathClamp from 'math-clamp';
 
+mathClamp(1, {min: 2, max: 4});
+//=> 2
+
 mathClamp(1, {minimum: 2, maximum: 4});
```
Comment (sindresorhus) [inline readme.md]: I don't want to document both. Only `min`.

Comment (Richienb) [reply readme.md]: Still going to include the longer form in the function signature part

File: index.js [hunk 2]
```diff
@@ -1,14 +1,30 @@
-export default function mathClamp(number, {minimum, maximum}) {
-	if (minimum > maximum) {
-		throw new RangeError('`minimum` should be lower than `maximum`');
+export default function mathClamp(number, {min, minimum, max, maximum}) {
+	if (minimum !== undefined) {
+		if (min === undefined) {
+			min = minimum;
+		} else {
+			throw new TypeError('`min` and `minimum` are mutually exclusive');
```
Comment (sindresorhus) [inline index.js]: I don't think this is nessecary. It should just be an alias. Just make `min` override `minimum`.

File: readme.md [hunk 3]
```diff
@@ -13,19 +13,19 @@ npm install math-clamp
...
 
-mathClamp(5, {maximum: 4});
+mathClamp(5, {max: 4});
 //=> 4
 ```
 
 ## API
 
-### mathClamp(number, {minimum?, maximum?})
+### mathClamp(number, {min/minimum?, max/maximum?})
```
Comment (sindresorhus) [inline readme.md]: I'm not going to document both. That just complicates it for the user. The intention is to just keep `minimum` as a hidden compatibility fallback.

File: index.js [hunk 4]
```diff
@@ -1,14 +1,17 @@
-export default function mathClamp(number, {minimum, maximum}) {
-	if (minimum > maximum) {
-		throw new RangeError('`minimum` should be lower than `maximum`');
+export default function mathClamp(number, {min, minimum, max, maximum}) {
+	min ??= minimum;
```
Comment (Richienb) [inline index.js:3]: Worth adding a comment about it being for back-compat, and for later removal?

Comment (sindresorhus) [reply index.js:3]: Yes

**Pull request merged**